crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/execution/entry_point.rs

```diff
@@ -35,10 +35,19 @@ pub fn execute_call_entry_point(
 ) -> EntryPointExecutionResult<CallInfo> {
     // region: Modified blockifier code
     // We skip recursion depth validation here.
-    runtime_state
-        .cheatnet_state
-        .trace_data
-        .enter_nested_call(entry_point.clone(), resources.clone());
+    let class_hash = if let Some(class_hash) = entry_point.class_hash {
+        class_hash
+    } else {
+        state
+            .get_class_hash_at(entry_point.storage_address)
+            .unwrap()
```

Comment:
Is class_hash for storage_address always present?

Reply:
I don't see a case where it is not present, we can always hotfix it if we figure out that there can be a contract address without a class hash

Reply:
So let's at least add a context 🙏

Reply:
Done